test/src/nyc-test.js

```diff
@@ -232,6 +248,39 @@ describe('nyc', function () {
       })
     })
 
+    describe('compile handlers for custom extensions are assigned', function () {
+      it('assigns a function to custom extensions', function () {
+        var nyc = new NYC({
+          cwd: process.cwd()
```

Comment:
Reference for my previous comment: https://github.com/bcoe/nyc/blob/master/index.js#L54